docs/editor/profiles.md

```diff
@@ -0,0 +1,95 @@
+---
+Order: 15
+Area: editor
+TOCTitle: Profiles
+ContentId: a65efc48-5a2d-4c7d-bd23-03f0393b53f6
+PageTitle: Profiles in Visual Studio Code
+DateApproved: 2/2/2023
+MetaDescription: Expand your development workflow with task integration in Visual Studio Code.
+---
+# Profiles in Visual Studio Code
+
+## Create a Profile
+
```

Comment:
I would love us to link from this doc to a profiles video @burkeholland is working on.
That should probably be at the top